src/daemon/emitter.cc

```diff
@@ -579,6 +608,27 @@ void Emitter::DoRemoteExecute(const base::WorkerPool& pool, ResolveFn resolver,
     failed_tasks_->Push(std::move(*task));
     counter.ReportOnDestroy(true);
   }
+
+  auto new_conf = this->conf();
```

Comment:
Pool gets shutted down only after a new pool(with new configuration) has started. So yeah, it should've beed reloaded by this time.